js/turtle.js

```diff
@@ -976,25 +976,33 @@ function Turtle (name, turtles, drum) {
         }
     };
 
-    this.blink = function() {
+    this.blink = function(duration) {
         var turtle = this;
+        var defaultsize = 55;
+        var newsize = 60;
         turtle.bitmap.alpha = 0.5;
+        turtle.bitmap.scaleX = Number(newsize) / defaultsize;
```

Comment:
Oops. You need to keep track of the old scale before you assign the new scale and then restore it afterward since some "reskinned" turtles are not at scale = 1.